test/__init__.py

```diff
@@ -1,22 +1,20 @@
-"""
-mbed SDK
-Copyright (c) 2011-2015 ARM Limited
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-"""
+# Copyright (c) 2021, Arm Limited and affiliates.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
This was preexisting: can we not just use the SPDX ID in the source, why do we have to copy the entire license text into every source file?

Reply:
Previously also I have received this review comment https://github.com/ARMmbed/greentea/pull/302#discussion_r640612505 to use `comments for the licence header`

Reply:
OK, we can add a task to remove the full text later, if we think it's important.